src/commands/profile.js

```diff
@@ -5,15 +5,40 @@ const colors = require('colors/safe');
 const Logger = require('../logger.js');
 const User = require('../models/user.js');
 
-async function profile () {
-  const { id, name, email, preferredMFA } = await User.getCurrent();
-  const has2FA = (preferredMFA != null && preferredMFA !== 'NONE') ? 'yes' : 'no';
-  const formattedName = name || colors.red.bold('[not specified]');
-  Logger.println('You\'re currently logged in as:');
-  Logger.println('User id          ' + id);
-  Logger.println('Name             ' + formattedName);
-  Logger.println('Email            ' + email);
-  Logger.println('Two factor auth  ' + has2FA);
+async function profile (params) {
+  const { format } = params.options;
+
+  const user = await User.getCurrent();
+
+  const formattedUser = {
+    id: user.id,
+    email: user.email,
+    name: user.name,
+    phone: user.phone,
+    address: user.address,
+    city: user.city,
+    zipcode: user.zipcode,
+    country: user.country,
+    creationDate: new Date(user.creationDate),
+    lang: user.lang,
+    emailValidated: user.emailValidated,
+    has2FA: (user.preferredMFA != null && user.preferredMFA !== 'NONE'),
+  };
+
+  switch (format) {
+    case 'json': {
+      Logger.printJson(formattedUser);
+      break;
+    }
+    case 'human':
+    default: {
+      Logger.println('You\'re currently logged in as:');
+      Logger.println(`User id          ${formattedUser.id}`);
+      Logger.println(`Name             ${formattedUser.name ?? colors.red.bold('[not specified]')}`);
+      Logger.println(`Email            ${formattedUser.email}`);
+      Logger.println(`Two factor auth  ${formattedUser.has2FA ? 'yes' : 'no'}`);
+    }
```

Comment:
Here I would add some infos (and adapt to change above):

```suggestion
      Logger.println(`User id          ${user.id}`);
      Logger.println(`Name             ${user.name ?? colors.red.bold('[not specified]')}`);
      Logger.println(`Email            ${user.email}`);
      Logger.println(`OAuth apps       ${user.oauthApps.join(', ')}`);
      Logger.println(`Two factor auth  ${user.has2FA ? 'yes' : 'no'}`);
      Logger.println(`Admin            ${user.admin ? 'yes' : 'no'}`);
      Logger.println(`Can pay?         ${user.canPay ? 'yes' : 'no'}`);
    }
```